alpine-consul/root/etc/services.d/consul-template/finish

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/execlineb -S1
+
+# only tell s6 to bring down the entire container, if it isn't already doing so
+# http://skarnet.org/software/s6/s6-supervise.html
```

Comment:
@matthewvalimaki, I think it would be better if `consul-template` automatically restarted rather than bringing down the container (to do this just remove this file). `consul-template` is a secondary service to consul itself and I think it would be fine to have s6 automatically restart it.

I think it would also go aways to resolving the `consul-template` runs before `consul` is ready issue (aside from the `retry` option I've just commented about).

What do you think?